station-worldgen-api-v0/src/main/java/net/modificationstation/stationapi/api/worldgen/BiomeAPI.java

```diff
@@ -0,0 +1,117 @@
+package net.modificationstation.stationapi.api.worldgen;
+
+import net.minecraft.level.biome.Biome;
+import net.modificationstation.stationapi.api.registry.Identifier;
+import net.modificationstation.stationapi.api.worldgen.biome.BiomeProvider;
+import net.modificationstation.stationapi.api.worldgen.biome.BiomeRegionsProvider;
+import net.modificationstation.stationapi.impl.worldgen.NetherBiomeProviderImpl;
+import net.modificationstation.stationapi.impl.worldgen.OverworldBiomeProviderImpl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BiomeAPI {
+	private static Map<Identifier, BiomeProvider> overworldProviders = new HashMap<>();
+	private static Map<Identifier, BiomeProvider> netherProviders = new HashMap<>();
```

Comment:
Reference2ObjectMap, potentially? Identifiers are cached and can be compared by just identity for faster lookup.

Reply:
And for that place map can be changed since this storage is not temporal, will do that

Reply:
Done